src/theme.tsx

```diff
@@ -458,13 +469,36 @@ export default mergeTheme(defaultTheme, {
           },
         },
       },
+      ({ destructive }: any) => destructive && {
+        color: 'text-error',
+        backgroundColor: 'transparent',
+        border: '1px solid border-error',
+        ':hover': {
+          backgroundColor: 'action-input-hover',
+          border: '1px solid border-error',
+        },
+        ':active': {
+          backgroundColor: 'transparent',
+          border: '1px solid border-error',
+        },
+        ':disabled': {
+          backgroundColor: 'transparent',
+          border: '1px solid border-error',
```

Comment:
like nick said, this should probably be the standard disabled border color

Reply:
Fixed.